packages/common/src/utils/validateSceneName.ts

```diff
@@ -31,7 +31,7 @@ import {
 
 export default function isValidSceneName(sceneName: string) {
   return (
-    sceneName.length >= 3 &&
+    sceneName.length >= 4 &&
```

Comment:
I was thinking, we can enforce all four of this conditions in the `VALID_SCENE_NAME_REGEX` itself.

`/^[a-zA-Z0-9][a-zA-Z0-9-]{2,62}[a-zA-Z0-9_\-]$/` (Updated `{1,62}` to `{2,62}`)

This checks all the conditions (allowed length, allowes only valid characters).

Reply:
nice. implemented.